src/markdownSyntaxHighlightOptions.js

```diff
@@ -21,9 +21,26 @@ module.exports = function(options = {}) {
       html = Prism.highlight(str, PrismLoader(language), language);
     }
 
+        //  Prism's markdown highlighter renders tables
+        //  with a newline before the end of the corresponding
+        //  markdown line. It's hacky, but I'd rather
+        //  fix it up here than figure out how to
+        //  fix Prism's markdown highlighter
+
+    if(language === 'markdown'){
+      let wickedPattern=/<\/span>\n<\/span>/g
+      let goodReplacement='</span></span>\n'
+      html = html.replace(wickedPattern, goodReplacement)
```

Comment:
Hmm—this block seems very risky to apply globally to all markdown code?

Reply:
My reasoning was this:

- it _is_ hacky & kind of lazy
 I can probably check to see whether we're in a `<table>` context
- the change
from `</tag>[WHITESPACE]</tag>`
to `</tag></tag>[WHITESPACE]`
seems to be neutral (whitespace doesn't matter in the HTML)
- the real problem is in Prism, but… I can fix this here more easily than in Prism. (for certain values of easy)